### ISSUE TITLE
[Android Editor] export loading indicator is missing

### ISSUE THREAD

**Issue by shahriarlabib000** [godotengine/godot#98019]
### Tested versions

- Reproducible in 4.4.dev3

### System information

Godot v4.4.dev3 - Android - Single-window, 1 monitor - OpenGL ES 3 (Compatibility) - Mali-G52 -  (8 threads)

### Issue description

AFAIK there was a loading indicator. The loading screen when exporting from the Android Editor seems to be missing same for when installing Export template. It feels like the Editor is unresponsive without the loading indicator

### Steps to reproduce

Just export anything project from the Android Editor 

### Minimal reproduction project (MRP)

N/A

**Comment by UnderGamer05** [comment]
Probably this is because of your device hardware.

**Comment by shahriarlabib000** [comment]
Could be but i think I saw the loading indicator in some version of godot, can't remember which one tho

**Comment by shahriarlabib000** [comment]
This isn't a problem in mobile renderer for some reason

**Comment by syntaxerror247** [comment]
I can reproduce this in 4.4.beta1

@shahriarlabib000 Can you please confirm this is a regression by trying older versions?

**Comment by shahriarlabib000** [comment]
> [@shahriarlabib000](https://github.com/shahriarlabib000) Can you please confirm this is a regression by trying older versions?

I don't think it's a regression. Reproducible in 4.4.dev2 when #93526 was added.

**Comment by llama-nl** [comment]
@syntaxerror247 I don't think it is a Device pesific. As far as I know Compatibility Renderer has not loading screen, but Mobile Renderer has.

**Comment by syntaxerror247** [comment]
> [@syntaxerror247](https://github.com/syntaxerror247) I don't think it is a Device pesific. As far as I know Compatibility Renderer has not loading screen, but Mobile Renderer has.

I can reproduce this issue with the Mobile Renderer as well.  

Based on my current tests:  
- **Compatibility Renderer:** The progress bar is not visible for any loading tasks.  
- **Mobile Renderer:** The progress bar is visible for editor loading, imports, and other tasks, but it does not appear for exporting or loading export templates.

**Comment by shahriarlabib000** [comment]
Mobile Renderer

https://github.com/user-attachments/assets/75412839-f1c6-49f8-974c-f6893aa24238

https://github.com/user-attachments/assets/73e4deaa-ff6d-4f59-a7e9-e7a3ee4cceb4

**Comment by shahriarlabib000** [comment]
So in mobile renderer it seems to be a regression

**Comment by syntaxerror247** [comment]
I suspect this issue is due to https://github.com/godotengine/godot/pull/99844
@shahriarlabib000 Can you test after reverting this PR?

**Comment by shahriarlabib000** [comment]
> I suspect this issue is due to [#99844](https://github.com/godotengine/godot/pull/99844) [@shahriarlabib000](https://github.com/shahriarlabib000) Can you test after reverting this PR?

Yep reverting that PR fixed the issue in Mobile Renderer.

To clarify the issue still stays in compatibility Renderer

**Comment by syntaxerror247** [comment]
@KoBeWi Can you please take a look at this issue.

**Comment by KoBeWi** [comment]
I can't reproduce it on Windows and I don't know what could be causing it. Do you see progress dialog when e.g. saving? The dialog will appear over currently focused Window, maybe it's not determined properly on Android. Try moving the dialog off the center (if possible); if progress appears, it will be behind the dialog.

**Comment by shahriarlabib000** [comment]
> if progress appears, it will be behind the dialog.

Yep,

![Image](https://github.com/user-attachments/assets/6e1bcbda-b5e6-42bf-9bf6-98120ab48a89)

**Comment by syntaxerror247** [comment]
@KoBeWi Android runs in single window mode. It must be a problem with that.
